dhis-2/dhis-web/dhis-web-commons-resources/src/main/webapp/dhis-web-commons/useraccount/restore.vm

```diff
@@ -50,8 +50,13 @@ You account was successfully restored. Use your new password to
 </div>
 
 <div id="restoreErrorMessage" style="display:none">
-Sorry, your account could not be restored. The code might be invalid or expired. Try again or
-go to the <a href="login.action">login page</a> and request a new restore email.
+Sorry, your account could not be restored. You might have chosen an invalid password.
+A password must be between $minPasswordLength and $maxPasswordLength characters and
+have at least one uppercase character, number, and symbol. It may not contain your
```

Comment:
In the spirit of making it less confusing, I just want to point out that we here talk about uppercase character, number and symbol, but in the other error message, capital letter, digit and special character. Maybe we can stick with one or the other :)

Reply:
Thanks, Stian. Point well taken and I have changed this to say capital letter like the Javascript message. Note that both "capital letter" and "uppercase character" are still used in various password validation messages in various places in DHIS2, something that perhaps we can resolve for future versions when we tackle the rest of https://jira.dhis2.org/browse/DHIS2-9065.